### PR TITLE
ap-fluentd: image should be created with a non root user

### PR DESCRIPTION
**Pull request by andriisoldatenko**
resolves https://github.com/astronomer/issues/issues/1290

### CODE REVIEW

**Comment by pgvishnuram** [comment]
can we move LABEL maintainer="Astronomer <humans@astronomer.io>" to top

**Comment by andriisoldatenko** [comment]
@pgvishnuram done.

**Review by pgvishnuram — APPROVED** [review]

File: fluentd/Dockerfile [hunk 1]
```diff
@@ -15,6 +15,11 @@
 
 FROM debian:sid-slim
 
+LABEL maintainer="Astronomer <humans@astronomer.io>"
+LABEL io.astronomer.docker=true
+LABEL io.astronomer.docker.build.number=$BUILD_NUMBER
+
+ARG BUILD_NUMBER=-1
```
Comment (vishwas-astro) [inline fluentd/Dockerfile:22]: shouldn't you define the BUILD_NUMBER before where it is used? See line 20.

**Comment by andriisoldatenko** [comment]
@vishwas-astro PTAL

File: fluentd/Dockerfile [hunk 2]
```diff
@@ -43,10 +48,19 @@ RUN set -o xtrace && \
         /var/cache/debconf/* \
         /var/lib/apt/lists/* \
         /var/log/* \
```
Comment (danielhoherd) [inline fluentd/Dockerfile:50]: Match indentation of newer lines
```suggestion
       /var/cache/debconf/* \
       /var/lib/apt/lists/* \
       /var/log/* \
```

**Review by danielhoherd — APPROVED** [review]
LGTM with one minor whitespace suggestion

**Comment by andriisoldatenko** [comment]
@danielhoherd PTAL

**Review by danielhoherd — APPROVED** [review]

**Pull request merged**